os/drivers/lcd/Kconfig

```diff
@@ -145,6 +145,48 @@ config LCD_RLANDSCAPE
 
 endchoice
 
+config LCD_ST7701
+        bool "ST7701 LCD Driver"
+        default n
+        ---help---
+                LCD Driver, ST7701.
+
+                Required LCD driver settings:
+                LCD_MAXPOWER should be 1:  0=off, 1=on
+
+config LCD_ST7701_XRES
+        int "ST7701 X Resolution"
+        default 480
+        ---help---
+                Specifies the X resolution of the LCD.
+
+config LCD_ST7701_YRES
+        int "ST7701 Y Resolution"
+        default 800
+        ---help---
+                Specifies the Y resolution of the LCD.
```

Comment:
They should be in LCD_ST7701 conditional.
```
if LCD_ST7701
...
endif
```
or
```
config LCD_7701_XRES
    ...
    depends on LCD_ST7701
```

Need to check ST7785, ST7789 as well.